### ISSUE TITLE
Lexmatch files have some duplicate lines

### ISSUE THREAD

**Issue by twhetzel** [monarch-initiative/mondo-ingest#591]
Duplicate lines in lexmatch files were identified in [this PR](https://github.com/monarch-initiative/mondo-ingest/pull/586#discussion_r1654081979).

**Comment by sabrinatoro** [comment]
It is expected that there are duplicated lines, because, for the same Mondo-resource matches, some lexmatch could be label to synonym, other synonym to synonym, etc.

**Comment by joeflack4** [comment]
True, but these are even more duplicative than that. The rows Trish highlighted are exactly the same; it isn't just that 1 row is like a label match and the other is a synonym.
